src/actions/websiteAuth.js

```diff
@@ -34,22 +38,11 @@ export function authenticate(username, password, rememberMe = false) {
     dispatch);
 }
 
-export function login(saveCookie) {
-  return (dispatch, getState) => {
-    if (saveCookie) {
-      siteCookie.save(getState().websiteAuth);
-    }
-    dispatch({
-      type: 'WEBSITE_AUTH_COMPLETE'
-    });
-  };
-}
-
 export function refresh() {
   return (dispatch, getState) => {
-    const { websiteAuth: { refreshToken }} = getState();
+    const { websiteAuth: { refresh_token }} = getState();
```

Comment:
If we do something like 
```suggestion
    const  { refresh_token } = siteCookie.get();
```
We don't even need to save website auth in the redux store.